README.md

```diff
@@ -0,0 +1,34 @@
+![github_banner](http://res.cloudinary.com/chi6rag/image/upload/v1431578923/HourGlass/hourglass_banner.png)
+
+HourGlass
+=========
+
+HourGlass lets you track the time you spend physically at a location.
```

Comment:
:+1:  Good job documenting the roadmap chirag!
We could potentially use this as a high level roadmap and keep high level todo list here.
You can checkoff things (visually) as you keep working on them.